code/modules/surgery/organs/organ_internal.dm

```diff
@@ -31,6 +31,10 @@
 	var/list/food_reagents = list(/datum/reagent/consumable/nutriment = 5)
 	///The size of the reagent container
 	var/reagent_vol = 10
+	///Stage of failing
+	var/failure_stage = 0
+	var/failure_time = 0
+	var/failure_progression_time = 60 //a minute
```

Comment:
```suggestion
	var/failure_progression_time = 60 SECONDS
```

Reply:
The current value of 60 will cause you to go up a stage every 6 seconds.

Reply:
don't think that's the case , i tested it and it is a minute

Reply:
yeah it is exactly 60 seconds because we add 2 to failure_time every 2 seconds

Reply:
there is no need to add SECONDS